src/cluster.c

```diff
@@ -5085,7 +5085,7 @@ void dumpCommand(client *c) {
 
 /* RESTORE key ttl serialized-value [REPLACE] */
 void restoreCommand(client *c) {
-    long long ttl, lfu_freq = -1, lru_idle = -1, lru_clock = -1;
```

Comment:
I prefer to keep `ttl` in the name as `milliseconds` is too generic and doesn't indicate this is the time to live. If you want to make it clear that the TTL is in terms of milliseconds, you can rename it to be something like `ttl_millis`.

Reply:
I dont feel strongly but existing code uses `milliseconds` already: https://github.com/redis/redis/blob/92bde124caaedef0e1572765854b2deb9aa22a35/src/t_string.c#L76

Reply:
A downside to changing it is it confuses the git history. I don't really care.

Reply:
I didn't take git history into account.

I can revert my ttl variable naming changes in this PR.

I still think it's worthwhile to unify how TTL variables are named across different places.

If we think `ttl` is better/sufficient. Should we want to change places like this too?

https://github.com/redis/redis/blob/92bde124caaedef0e1572765854b2deb9aa22a35/src/t_string.c#L76

Reply:
In general, we don't want to change small things like these, except if it really improves the code. If there is no risk of confusion (or only a very small risk), I suggest keeping the old variable names.